package.json

```diff
@@ -56,10 +56,12 @@
     "gulp-sourcemaps": "~1.5.2",
     "gulp-util": "~2.2.14",
     "jest": "^20.0.4",
+    "minimist": "^1.2.0",
```

Comment:
I'd rather not introduce new dependencies if possible – if we do keep the gulp integration, could we pass the arguments to stylelint directly instead of adding another layer of indirection, and dependency?